tests/clv/models/test_pareto_nbd.py

```diff
@@ -338,8 +433,8 @@ def test_posterior_distributions(self, fake_fit, T) -> None:
         assert customer_purchase_rate.shape == expected_shape
         assert customer_rec_freq.shape == expected_pop_dims
 
-        lam = pm.Gamma.dist(alpha=self.r_true, beta=1 / self.alpha_true, size=N)
-        mu = pm.Gamma.dist(alpha=self.s_true, beta=1 / self.beta_true, size=N)
+        lam = pm.Gamma.dist(alpha=self.r_true, beta=self.alpha_true, size=N)
+        mu = pm.Gamma.dist(alpha=self.s_true, beta=self.beta_true, size=N)
```

Comment:
Why this change?

Reply:
The beta parameters weren't specified properly. More details here:
https://github.com/pymc-labs/pymc-marketing/issues/328